src/Toolkit/kits/shadcn/dropdown-menu/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "../../../schema-kit-recipe-v1.json",
+    "type": "component",
+    "name": "Dropdown Menu",
+    "description": "A menu triggered by a button, providing a list of actions or links.",
+    "copy-files": {
+        "assets/": "assets/",
+        "templates/": "templates/"
+    },
+    "dependencies": {
+        "recipe": ["kbd"],
```

Comment:
Let's remove this line, as kbd is not required to render the dropdown menu

Reply:
Done.